packages/react-components/react-rating/library/src/components/Rating/Rating.test.tsx

```diff
@@ -51,4 +51,12 @@ describe('Rating', () => {
     expect(onChange.mock.calls[1][1].value).toBe(3);
     expect(onChange.mock.calls[2][1].value).toBe(2);
   });
+  it('creates RadioItems with correct aria-labels', () => {
+    const onChange = jest.fn();
+    const { getAllByRole } = render(<Rating onChange={onChange} itemLabel={(num: number) => `item #${num}`} />);
```

Comment:
nit: the prop is already typed, so you could strip the types there.

```suggestion
    const { getAllByRole } = render(<Rating onChange={onChange} itemLabel={(num) => `item #${num}`} />);
```

Reply:
Applied suggestion